showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITPagination.java

```diff
@@ -0,0 +1,165 @@
+package com.google.showcase.v1beta1.it;
```

Comment:
Do we need a copyright header for this class?

Reply:
Yep, added.